### ISSUE TITLE
Closed Captions Support

### ISSUE THREAD

**Issue by shatteredsite** [videojs/videojs-youtube#564]
Right now I do not see closed captions support within the plugin. This is essential for Americans With Disabilities Act compliance, which is now being applied by law to websites in many instances. It is also important for many businesses.

**Comment by gkatsev** [comment]
Yeah, it would be nice to have. Unfortunately, it doesn't seem like the [youtube iframe api](https://developers.google.com/youtube/iframe_api_reference) provides a list of available captions or a way to select one.

**Comment by arnoldsimha** [comment]
@gkatsev There is a way to get it from what I see:
https://stackoverflow.com/a/58435817/2339631

**Comment by gkatsev** [comment]
While that is an official Google API, that doesn't seem to be part of the youtube iframe API, and so we're unlikely to implement it into videojs-youtube. However, it's definitely possible to write a Video.js plugin that's used in conjunction with videojs-youtube to grab these caption files.
